frontend/lib/src/components/input/search_field.dart

```diff
@@ -6,9 +6,14 @@ import 'package:flutter_form_builder/flutter_form_builder.dart';
   Similar to InputField, but this component adds the icon and the hint text.
 */
 class SearchField extends StatelessWidget {
-  const SearchField({super.key, required this.name, this.validator});
-
+  const SearchField({
+    super.key,
+    required this.name,
+    this.validator,
```

Comment:
add @param descriptions i above comments for
validator (if InputField doesnt have it yet)
and yPaddings